schema/v1/trigger-event.json

```diff
@@ -0,0 +1,188 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Trigger Event",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string"
+    },
+    "id": {
```

Comment:
@eacarvalho why do we have to provide an id if the eventClassName would be unique. Just to have a shorter PK?

Reply:
Yes and this is unique, the `eventClassName` might change during update process. So basically `id` is the key to identify if it is a creation or needs to update an event general notification definition.